components/tools/ToolsCard.tsx

```diff
@@ -1,5 +1,4 @@
 import { useEffect, useRef, useState } from 'react';
-import TextTruncate from 'react-text-truncate';
```

Comment:
`TextTruncate` library caused layout shifts post-render, preventing consistent scroll padding. Removing it enabled uniform padding across tools page elements.